source/usage-examples/count.txt

```diff
@@ -9,14 +9,17 @@ collection by using the ``EstimatedDocumentCount()`` method and an exact
 number of documents in a collection by using the ``CountDocuments()``
 method.
 
-The following example uses the ``EstimatedDocumentCount()`` method to
-count the number of documents in the ``movies`` collection. Then, the
-example uses the ``CountDocuments()`` method to count the number of
-documents in the ``movies`` collection where the ``countries`` field
-contains "China".
+Example
+-------
 
 .. include:: /includes/usage-examples/run-example-tip.rst
 
+The following example uses the ``EstimatedDocumentCount()`` method to
+count the number of documents in the ``movies`` collection. Then, the
+example passes a query filter to the ``CountDocuments()`` method, which
+counts the number of documents in the ``movies`` collection where the
+``countries`` field contains "China".
```

Comment:
This seems like a lot of information for a single sentence. I wonder if you might consider using a bulleted list or at least separate sentences instead?

Applies to all instances.